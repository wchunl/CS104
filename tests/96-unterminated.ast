ROOT "" (0.0.0)
|   TOK_VARDECL "=" (4.6.13)
|   |   TOK_STRING "string" (4.6.4)
|   |   |   TOK_TYPE_ID "s" (4.6.11)
|   |   TOK_IDENT "abc" (4.6.16)
|   '=' "=" (4.8.6)
|   |   TOK_IDENT "s" (4.8.4)
|   |   TOK_IDENT "abcd" (4.8.9)
|   '=' "=" (4.9.6)
|   |   TOK_IDENT "s" (4.9.4)
|   |   TOK_IDENT "abc" (4.9.9)
