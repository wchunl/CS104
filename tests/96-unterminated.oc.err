96-unterminated.oc:6:15: warning: missing terminating " character [enabled by default]
    string s = "abc;
               ^
96-unterminated.oc:7:13: warning: missing terminating ' character [enabled by default]
    char c = 'a;
             ^
96-unterminated.oc:8:8: warning: missing terminating " character [enabled by default]
    s = "abcd\";
        ^
96-unterminated.oc:9:8: warning: missing terminating " character [enabled by default]
    s = "abc|\
        ^
Hg›:5.15: invalid source character (")
Hg›:5.16: invalid source character (\)
Hg›:5.17: syntax error, unexpected '*'
Hg›:5.19: invalid source character (")
Hg›:6.15: invalid source character (")
Hg›:7.9: syntax error, unexpected TOK_IDENT
Hg›:7.13: invalid source character (')
Hg›:8.8: invalid source character (")
Hg›:8.13: invalid source character (\)
Hg›:8.14: invalid source character (")
Hg›:9.8: invalid source character (")
Hg›:9.12: invalid source character (|)
Hg›:11.8: invalid token (23foobar)
Hg›:11.8: syntax error, unexpected $undefined, expecting TOK_IDENT
