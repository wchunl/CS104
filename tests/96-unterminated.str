string_set[   1]:     831242270113464010 0x9b6bd8->"t"
string_set[   2]:   11868120863286193964 0x9b25a8->"main"
                   13497578857207247412 0x9b6b38->"string"
string_set[   3]:    3663726644998027833 0x9b6f38->"abc"
                    5344662657899890615 0x9b6e78->";"
                    6637313742931709005 0x9b6d38->"*"
                   11597697714117577063 0x9b6c98->"="
string_set[   4]:   10959529184379665549 0x9b7018->"c"
string_set[   9]:     342363981814211588 0x9b6fc8->"char"
string_set[  11]:      21606623943142546 0x9b6a98->"{"
string_set[  12]:   13686721134187162920 0x9b6f88->"}"
                    7300644294671580816 0x9b7108->"23foobar"
string_set[  14]:   16030370620903852840 0x9b70b8->"abcd"
                    6414370376350317282 0x9b6ec8->"s"
                   12868445110721718657 0x9b69c8->")"
                   15816506932648033945 0x9b2228->"("
                    6142509188972423790 0x9b2588->""
string_set[  15]:   12865648880272446602 0x9b6bf8->"/"
string_set[  17]:    4993892634952068459 0x9b7068->"a"
string_set[  21]:   12597418120160561823 0x9b6688->"int"
load_factor = 0.870
bucket_count = 23
max_bucket_size = 5
