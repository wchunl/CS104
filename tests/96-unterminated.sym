
s (4.6.4)  string variable lval 0
