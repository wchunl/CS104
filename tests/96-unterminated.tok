#  1 "96-unterminated.oc"
#  1 "<built-in>"
#  1 "<command-line>"
#  1 "96-unterminated.oc"
   4 4.00001 262 TOK_INT       int
   4 4.00005 282 TOK_IDENT     main
   4 4.00009  40 '('           (
   4 4.00010  41 ')'           )
   4 4.00012 123 '{'           {
   4 5.00004 263 TOK_STRING    string
   4 5.00011 282 TOK_IDENT     t
   4 5.00013  61 '='           =
   4 5.00017  42 '*'           *
   4 5.00018  47 '/'           /
   4 5.00020  59 ';'           ;
   4 6.00004 263 TOK_STRING    string
   4 6.00011 282 TOK_IDENT     s
   4 6.00013  61 '='           =
   4 6.00016 282 TOK_IDENT     abc
   4 6.00019  59 ';'           ;
   4 7.00004 282 TOK_IDENT     char
   4 7.00009 282 TOK_IDENT     c
   4 7.00011  61 '='           =
   4 7.00014 282 TOK_IDENT     a
   4 7.00015  59 ';'           ;
   4 8.00004 282 TOK_IDENT     s
   4 8.00006  61 '='           =
   4 8.00009 282 TOK_IDENT     abcd
   4 8.00015  59 ';'           ;
   4 9.00004 282 TOK_IDENT     s
   4 9.00006  61 '='           =
   4 9.00009 282 TOK_IDENT     abc
   4 9.00016  59 ';'           ;
   4 11.00004 262 TOK_INT       int
   4 11.00008  50 $undefined    23foobar
   4 11.00016  59 ';'           ;
   4 12.00001 125 '}'           }
