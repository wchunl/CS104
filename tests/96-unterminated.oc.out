; 0.  1: # 1 "96-unterminated.oc"; 1.  0: 
; 1.  1: # 1 "<built-in>"
; 2.  1: # 1 "<command-line>"
; 3.  1: # 1 "96-unterminated.oc"
; 4.  1: 
; 4.  2: 
; 4.  3: 
; 4.  4: int main() {
; 4.  5:    string t = "\*/";
; 4.  6:    string s = "abc;
; 4.  7:    char c = 'a;
; 4.  8:    s = "abcd\";
; 4.  9:    s = "abc|   ;
; 4. 10: 
; 4. 11:    int 23foobar;
; 4. 12: }
hello